supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala

```diff
@@ -100,7 +100,7 @@ sealed trait ConfigAsyncJobExecutionActor extends SharedFileSystemAsyncJobExecut
     val allInputs = providedWomInputs ++ optionalsForciblyInitializedToNone
     val womInstantiation = taskDefinition.instantiateCommand(allInputs, NoIoFunctionSet, identity, runtimeEnvironment)
 
-    val InstantiatedCommand(command, _, _, _) = womInstantiation.toTry.get
+    val InstantiatedCommand(command, _, _, _, _, _) = womInstantiation.toTry.get
```

Comment:
```scala
val command = womInstantiation.toTry.get.commandString
```

Reply:
I'm sure this made more sense at one point... 😦